A-Star/AStar.swift

```diff
@@ -54,10 +54,7 @@ public final class AStar<G: Graph> {
     /// - Precondition: both `source` and `target` belong to `graph`.
     public func path(start: G.Vertex, target: G.Vertex) -> [G.Vertex] {
         open.insert(Node<G.Vertex>(vertex: start, cost: 0, estimate: heuristic(start, target)))
-        while !open.isEmpty {
-            guard let node = open.remove() else {
-                break
-            }
+        while !open.isEmpty, let node = open.remove() {
```

Comment:
for better readability, please split `!open.isEmpty` and `let node = open.remove()` into two lines.

Reply:
I actually like this change. 😄 Having it all on one line would be my preference.